### DISCUSSION TITLE
How to add grid in first page of paging(Full Map Page)

### DISCUSSION THREAD

**Discussion by sureshenercomp** [mapfish/mapfish-print#3267] [General]
How to add grid in first page of paging(Full Map Page) so that user have an idea about sub pages of map on first page

**Comment by sbrunner** [comment]
With this example https://github.com/mapfish/mapfish-print/blob/master/examples/src/test/resources/examples/paging/config.yaml
you can have an overview map but without a grid...

**Reply by sureshenercomp** [reply]
i already using it but my client require grid on first page.i wanted to add a grid, where should i make changes if you could guide me

**Reply by sbrunner** [reply]
You want a grid on the map or a textual grid?
You take a look at the map page output https://github.com/mapfish/mapfish-print/blob/master/core/src/main/java/org/mapfish/print/processor/map/CreateMapPagesProcessor.java#L164-L169
Yon can also ask us for an offer: https://www.camptocamp.com/fr/lieux

**Reply by sureshenercomp** [reply]
how to add textual grid on full-map page of sub-map pages,only name of sub-map  would be ok
